app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -51,6 +57,19 @@ public void goToHomeScreen() {
         startActivity(intent);
     }
 
+    @Override
+    public void goToSignUpScreen() {
+        Intent intent = new Intent(getActivity(), SignUpActivity.class);
```

Comment:
This is perfect. We can add a good practice here. You can create a `start()` static final function in SignUpActivity with this code. So here you do `SignUpActivity().start()` to start the new activity. We can do this when you call LoginActivity and SignUpActivity in HomeActivity too.